fiftyone/server/routes/tagging.py

```
@@ -53,37 +62,258 @@ async def post(self, request: Request, data: dict) -> dict:
+                return result[0] if result else {}
+
+            results = await asyncio.gather(
+                *[
+                    run_slice(all_bounds[i], all_bounds[i + 1])
+                    for i in range(len(all_bounds) - 1)
+                ]
+            )
+
+        all_tags = set()
+        for doc in results:
+            for key, value in doc.items():
```

Ruff (0.15.9)

[warning] Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)